L/L_BFGS_B/build_tarballs.jl

```diff
@@ -3,17 +3,19 @@
 using BinaryBuilder
 
 name = "L_BFGS_B"
-version = v"3.0.0"
+version = v"3.0.1"
```

Comment:
The upstream source is always the same, why changing this version number?

Reply:
I think this is just the _jll package version number. Is this rule a convention or a mandatory limitation?

Actually, the patch should be submitted into the upstream, but the upstream seems to be not actively maintained. I bumped the patch number to notify users that this _jll package is a patched version of the upstream.

Reply:
> I think this is just the _jll package version number. Is this rule a convention or a mandatory limitation?

No, we use the same version number as the upstream package